include/eve/traits/as_floating_point.hpp

```diff
@@ -9,11 +9,16 @@
 
 #include <eve/detail/meta.hpp>
 #include <eve/detail/wide_forward.hpp>
+#include <eve/traits/translation.hpp>
 
 namespace eve
 {
   template<typename T>
-  struct as_floating_point : detail::make_floating_point<(sizeof(T) <= 4) ? 4: sizeof(T)>
+  inline constexpr auto under_size = sizeof(translate_element_type_t<T>);
```

Comment:
under size sounds like an action. size of trans;ated is always the same, you don't need this

Reply:
That's a remnant of before I forgot to remove. Will cleanup

Reply:
Fixed